luigi/file.py

```diff
@@ -83,6 +91,12 @@ def remove(self, path, recursive=True):
         else:
             os.remove(path)
 
+    def rename(self, *args, **kwargs):
+        """
+        Calls ``move()``
+        """
+        self.move(*args, **kwargs)
+
```

Comment:
Since this seems to be the same implementation for all `rename()` methods here. Why don't you make this the default behavior for `target.FileSystem.rename()` and not worry about rewriting the same thing for each class?